src/components/DisplayNames/DisplayNamesTooltipItem.tsx

```diff
@@ -62,18 +57,21 @@ function DisplayNamesTooltipItem({index, getTooltipShiftX, accountID, avatar, lo
             <Text
                 eslint-disable-next-line
                 no-param-reassign
-                // eslint-disable-next-line no-param-reassign
-                ref={(el) => (childRefs.current[index] = el)}
-                style={[...textStyles, styles.pre]}
+                ref={(el) => {
+                    if (!childRefs.current?.[index] || !el) {
+                        return;
+                    }
+                    // eslint-disable-next-line no-param-reassign
+                    childRefs.current[index] = el;
```

Comment:
I wonder if we can remove ` eslint-disable-next-line no-param-reassign` 
Can You try make a copy of childRef.currrent, modify this copy's elemnt by index and reassign whole ref?

Reply:
@barttom Since `childRefs` is a param passed to the `DisplayNamesTooltipItem` component, even if I'll try to reassign it all it gives the same eslint warning.

Reply:
Thanks for checking that!